### PR TITLE
3573 SYSTEST - Changes to federal-provincial-territorial-benefits

### PR DESCRIPTION
**Pull request by Fbasham**
### Description
Changes to text to match figma.

### Related Azure Boards Work Items
[AB#3573](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3573)

### CODE REVIEW

File: frontend/public/locales/fr/access-to-governmental-benefits.json
Comment (sebastien-comeau) [inline frontend/public/locales/fr/access-to-governmental-benefits.json:1]: Is that file still required?

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: frontend/public/locales/fr/apply-adult-child.json [hunk 1]
```diff
@@ -190,8 +190,8 @@
     "federal-benefits": {
       "title": "Prestations fédérales",
       "legend": "Bénéficiez-vous d'une assurance dentaire dans le cadre d'un programme social offert par un programme fédéral?",
-      "option-yes": "<strong>Oui</strong>, je bénéficie d'une assurance <strong>fédérale</strong>",
-      "option-no": "<strong>Non</strong>, je ne bénéficie pas d'une assurance <strong>fédérale</strong>",
+      "option-yes": "<strong>Oui</strong>, je bénéficie d'une assurance <strong>fédérale</strong> dentaires",
+      "option-no": "<strong>Non</strong>, je ne bénéficie pas d'une assurance <strong>fédérale</strong> dentaires",
```
Comment (sebastien-comeau) [inline frontend/public/locales/fr/apply-adult-child.json:194]: @yannismassyl said to move `dentaires` before `<strong> and it's `dentaire` with no `s`. Do the same in other places.

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: frontend/public/locales/fr/access-to-governmental-benefits.json
Comment (Fbasham) [reply frontend/public/locales/fr/access-to-governmental-benefits.json:1]: no clue.  It's used elsewhere in the app under the protected space.  Guess I'll just revert these changes since I basically just did a quick copy/paste everywhere I could

Comment (sebastien-comeau) [reply frontend/public/locales/fr/access-to-governmental-benefits.json:1]: Right, I didn't thought about this one.

**Review by sebastien-comeau — APPROVED** [review]
